Form: Add getScrollable Description (#8503)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxCardView/3 Methods/getScrollable().md

---
---
##### shortDescription
Returns the scrolling component within the CardView.

##### return: dxScrollable

> The scrolling component's instance (Scrollable).

Reviewer: So what is a `Scrollable`? I see that our component is named `ScrollView`. And you direct customers to use that component's API. But then what do you refer to when you say `Scrollable`? Why should we use that name... and should we?

Author: Scrollable is an internal component that our ScrollView is based on. ScrollView extends Scrollable's options with a handful of properties for standalone use. I think we should mention the name of the Scrollable mainly because it is used in the name of these methods. The Scrollable name can also be found in component markup, in class names such as `dx-scrollable`.

We can avoid mentioning the name of this component, but I think it is useful in the `Form.getScrollable` description. In the note of this description, using two occurrences of "Scrollable" instead of "scrolling component" is a touch more concise.

> The scrolling component's instance (Scrollable).

Reviewer: I guess we can then add a few details so that it's all clear. 

```suggestion
The scrolling component's instance (a `Scrollable` object).
```

---

> To configure Scrollable, use [ScrollView](/Documentation/ApiReference/UI_Components/dxScrollView/) configuration. The following options are not available:

Reviewer: ```suggestion
This method returns a `Scrollable` object used in our `ScrollView` component. As such, once you obtain the object, you can use [ScrollView](/Documentation/ApiReference/UI_Components/dxScrollView/) configuration properties. Note that the following options are not available:
```

> This method returns a `Scrollable` object used in our `ScrollView` component. As such, once you obtain the object, you can use [ScrollView](/Documentation/ApiReference/UI_Components/dxScrollView/) configuration properties. Note that the following options are not available:

Reviewer 2: ```suggestion
This method returns a `Scrollable` object used in our `ScrollView` component. After you obtain this object, you can use [ScrollView](/Documentation/ApiReference/UI_Components/dxScrollView/) configuration properties. Note that the following options are not available:
```
